### DISCUSSION TITLE
Week 2 - Instructional Tasks

### DISCUSSION THREAD

**Discussion by CraigButikofer** [USU-ITLS-x265-23-summer/class-repo#137] [Week 2 - Discussions]
Exercism and the FreeCodeCamp's instructional approach:

1. What I liked:
- The step by step instructions
- The ability to check the code instantly
2. What I disliked: 
- The instructions (or lack of example)
- Not getting to repeat the example with a different problem
3. What you would change:
- I would have a brief instructional video before you had to do anything
- I would have a second problem to solve to practice the coding

**Comment by beto0890** [comment]
Great Ideas!
